09_2024_scikit_learn_priorities.md

```diff
@@ -0,0 +1,256 @@
+   This new API consistently handles metadata (e.g., `sample_weight`, `groups`, etc.)
+   for all estimators. It allows for fewer bugs and enables new
+   use cases that were not possible before.
+
+Both of these projects represent major milestones for `scikit-learn`. In our current
+round of priorities, we outline follow-up work for both the website and metadata
+routing to further improve and expand on these changes.
+
+#### Mid-term priorities
+
+From the mid-term priorities, we successfully completed the following:
+
+1. Implementation of a new `scikit-learn` estimator, called `TunedThresholdClassifierCV`,
+   allowing optimization of the operational decision.
+1. Progress on the Array-API adoption by making it possible to train and test a pipeline
+   containing a `PCA` and `Ridge` estimator trained on `torch` tensors located on GPU.
+1. Some work to improve the developer experience by improving the developer tools
+   available in `scikit-learn` and made available for third-party projects.
+
+#### Long-term priorities
+
+In this last section, we report on long-term and sometimes recurring efforts that are
+important for the project's health.
+
+1. General maintenance of the project: We reviewed a substantial number of pull requests
+   and issues opened by external contributors.
+1. Python 3.13 support: We worked on supporting Python 3.13 free-threaded mode and set
+   up regular testing to ensure compatibility until it becomes officially available.
+1. Releases: Since February 2024, we released `scikit-learn` 1.5.0, 1.5.1, and 1.5.2.
+1. Continuous Integration and Continuous Deployment: We kept our continuous
+   integration and continuous deployment infrastructure up to date.
+1. Community outreach: Since February 2024, we attended multiple Python conferences
+   to promote the latest work on `scikit-learn`. We attended the following conferences:
+   PyCon Lithuania, PyCon Italia, CZI Open Science, EuroSciPy, PyData Amsterdam, and
+   PyData Paris.
+
+### Computing orchestration
+
+We dedicated resources to maintain the following projects that are related to computing
+orchestration: `joblib`, `loky`, `cloudpickle`, `threadpoolctl`, and support for
+Python free-threaded mode.
+
+`threadpoolctl` has been released in the past few months and includes better support
+for BLAS libraries (FlexBLAS, OpenBLAS, Netlib, Accelerate, etc.).
+
+We dedicated substantial work to ensuring scikit-learn's compatibility with Python 3.13's
+free-threaded mode, including extensive testing, necessary adaptations, and reporting
+any issues to upstream projects.
+
+### [`fairlearn`](https://github.com/fairlearn/fairlearn)
+
+In the past few months, the main focus of the project has been to ensure that the
+estimators developed in `fairlearn` are compatible with `scikit-learn`. To achieve this,
+we used the testing framework provided by `scikit-learn` to test the `fairlearn`
+estimators.
+
```

Comment:
Could also mention that we also pushed out a release with many compatibility fixes with dependency libraries.